nixos/release-small.nix

```diff
@@ -99,6 +99,7 @@ rec {
       mariadb
       nginx
       nodejs
+      xz-after-bootstrap
```

Comment:
Naturally, I wouldn't define this package globally, but only here via `callPackage`.

Reply:
makes complete sense yes.